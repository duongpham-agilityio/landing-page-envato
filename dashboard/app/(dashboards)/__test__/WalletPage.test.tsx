// Pages
import MyWalletPage from '../my-wallets/page';

const { render } = testLibReactUtils;

describe('MyWalletPage render', () => {
  test('Should render match with snapshot.', () => {
    const { container } = render(<MyWalletPage />);

    expect(container).toMatchSnapshot();
  });
});
