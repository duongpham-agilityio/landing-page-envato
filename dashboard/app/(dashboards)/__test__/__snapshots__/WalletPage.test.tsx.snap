// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard render Should render match with snapshot. 1`] = `
<div>
  <div
    class="css-et9gjv"
  >
    <div
      class="css-zd4of5"
    >
      <div
        class="css-6iskwu"
      />
    </div>
    <div
      class="css-gca1gs"
    >
      <div
        class="css-ahthbn"
      >
        <div
          class="css-hctu2c"
        >
          <div
            class="css-1n1b5z6"
          />
          <div
            class="css-ugfeub"
          />
        </div>
        <div
          class="css-0"
        >
          <section
            class="css-1gg9e9z"
          />
        </div>
      </div>
    </div>
  </div>
  <span
    hidden=""
    id="__chakra_env"
  />
</div>
`;
